.buildkite/caches.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "display_name": "npm cache",
+    "folder_to_archive": ".npm",
+    "folder_to_archive_basedir": "/root",
```

Comment:
I originally made the archives with `tar` manually, and it sort of helped to have the path split. But maybe it'd be neater to have the full path and use `dirname` or similar.